README/ReleaseNotes/v612/index.md

```diff
@@ -112,32 +112,46 @@ large TClonesArray where each element contains another small vector container.
   (the absolute value of MaxVirtualSize indicates how many additional clusters will be kept in memory).
 
 ### TDataFrame
-  - Improved documentation
-  - Fixed race condition: concurrent deletion of TTreeReader/TTreeReaderValue
-  - TDF now avoids performing virtual calls for parts of the analysis that are not jitted
-  - Improved checks for column name validity (throw if column does not exist and if `Define`d column overrides an already existing column)
-  - Removed "custom column" nodes from the functional graph therewith optimising the traversal
-  - Added `DefineSlot`, a `Define` transformation that is aware of the multi-threading slot where the workload is executed
-  - Improvements in Cling drastically enhanced scaling and performance of TDF jitted code
-  - Fixed reading of c-style arrays from jitted transformations and actions
-  - pyROOT users can now easily specify parameters for the TDF histograms and profiles thanks to the newly introduced tuple-initialization
-  - The new TDataSource interface allows developers to pipe any kind of columnar data format into TDataFrame
-  - Test coverage has been increased with the introduction of google tests
-  - Users can now configure Snapshot to use different file open modes ("RECREATE" or "UPDATE"), compression level, compression algorithm, TTree split-level and autoflush settings
-  - Python tutorials show the new "tuple-initialisation" feature of PyROOT (see below)
-  - The possibility to read from data sources was added. An interface for all data sources, TDataSource, is provided by ROOT. Two example data sources have been provided too: the TRootDS and the TTrivialDS. The former allows to read via the novel data source mechanism ROOT data, while the latter is a simple generator, created for testing and didactic purposes. It is therefore now possible to interface *any* kind of dataset/data format to ROOT as long as an adaptor which implements the pure virtual methods of the TDataSource interface can be written in C++.
-  - Column can be aliased with the `TDF::TInterface` method `Alias`: `auto histo = mytdf.Alias("myAlias", "myColumn").Histo1D("myAlias");`
-  - Add the `GetColumnsNames` method to the `TDF::TInterface`: the user can therefore get the names of the available columns coming from trees, data sources or `Define`d columns
-  - `TDataFrame`s can be cached in memory with the `TDF::TInterface` method `Cache`. All or some columns can be cached. Two versions of the method are proposed: one which allows to explicitly list the types of the columns and another one allowing to let the system infer them (the same mechanism of the `Snapshot` method). Only columns containing instances of classes which have a copy constructor can be cached.
+
+#### New features
+  - Add `Alias`, a facility to specify an alternative name for a given column: `auto histo = mytdf.Alias("myAlias", "myColumn").Histo1D("myAlias");`
```

Comment:
Maybe give an example of why it's useful? Like creating an alias for a column named `1branch` to `onebranch` to avoid problems with identifiers in Python, for example.